### ISSUE TITLE
Bug Report: Image Source

### ISSUE THREAD

**Issue by Yass32** [PalisadoesFoundation/talawa-docs#404]
**Describe the bug**
There seems to be a problem with the image source, it's not displaying

**Expected behavior**
An image

**Actual behavior**
Alt text being displayed

**Screenshots**
![Screenshot (27)](https://user-images.githubusercontent.com/108995581/224053138-6722a562-22f4-4679-9789-76aaf7cca4fc.png)

**Additional details**

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

Id like to work on this

**Comment by akankshat05** [comment]
Can i fix this ?

**Comment by SalilHarit** [comment]
Can we collaborate on this ?

**Comment by Haywayaheadshot** [comment]
@Cioppolo14 Can I be assigned to this issue?

**Comment by Yass32** [comment]
@Cioppolo14 id like to fix this

**Comment by tobidosumu** [comment]
@Yass32 Hello sir. Please assign me to fix this issue. Thanks.

**Comment by iamatifmoin** [comment]
@Cioppolo14 @palisadoes 
I have resolved this issue.

Before:
![before](https://user-images.githubusercontent.com/97212195/224745156-317e9f50-7554-452a-a4a4-f9f545973abd.png)

After:
![after](https://user-images.githubusercontent.com/97212195/224745185-fe139604-7a47-452b-90d3-5e7554929aff.png)

Should I make a PR for it?

**Comment by evasharma12** [comment]
> @Cioppolo14 @palisadoes I have resolved this issue.
> 
> Before: ![before](https://user-images.githubusercontent.com/97212195/224745156-317e9f50-7554-452a-a4a4-f9f545973abd.png)
> 
> After: ![after](https://user-images.githubusercontent.com/97212195/224745185-fe139604-7a47-452b-90d3-5e7554929aff.png)
> 
> Should I make a PR for it?

If the issue is resolved, make a PR

**Comment by iamatifmoin** [comment]
changing the file path from `assets/images/talawa-logo-lite-200x200.png` to `../../img/logos/talawa-logo-200x200.png` fixes the issue.

When I open the page through `Live Server` and inspect the image, the new path appears.
However, when I start the website using `yarn start run` the path is not changed.

How do I fix this?

**Comment by palisadoes** [comment]
@iamatifmoin As you claim to have fixed this, I've assigned you the issue. Please create a linked PR

**Comment by tasmeomeem** [comment]
can i fix it?

**Comment by SiddheshKukade** [comment]
@palisadoes Sir, I would like to get assigned to fix this.

**Comment by palisadoes** [comment]
Unassigning due to inactivity

**Comment by palisadoes** [comment]
You'll need to work with @anwersayeed @evasharma12 and @tasneemkoushar .

They are working on this automation

**Comment by SiddheshKukade** [comment]
Okay, I'll tag them to review the code once it's done. 👍🏻

**Comment by SiddheshKukade** [comment]
@anwersayeed, @evasharma12 , @tasneemkoushar I tried to do changes locally, but I can't check the output because the `docs` website just redirects me here: https://docs.talawa.io/talawa/index.html

Should I change the link from this file: https://github.com/PalisadoesFoundation/talawa/#readme 
from relative to absolute import should fix this according to me.
Am I correct, I think the HTML from https://docs.talawa.io/talawa/index.html is generated from [README.md](https://github.com/PalisadoesFoundation/talawa/#readme) right ?

**Comment by KinetiCore-Labs** [comment]
> @anwersayeed, @evasharma12 , @tasneemkoushar I tried to do changes locally, but I can't check the output because the `docs` website just redirects me here: https://docs.talawa.io/talawa/index.html
> 
> Should I change the link from this file: https://github.com/PalisadoesFoundation/talawa/#readme from relative to absolute import should fix this according to me. Am I correct, I think the HTML from https://docs.talawa.io/talawa/index.html is generated from [README.md](https://github.com/PalisadoesFoundation/talawa/#readme) right ?

Yes, it may fix the error

**Comment by SiddheshKukade** [comment]
Looks like the path is still hasn't been updated on https://docs.talawa.io/talawa/index.html
![image](https://user-images.githubusercontent.com/65951872/228819244-abccce76-8efe-4fb4-9dde-26413ce561ae.png)
What do we have to do to regenerate that HTML? 
cc @anwersayeed , @evasharma12, @tasneemkoushar

**Comment by palisadoes** [comment]
GitHub actions were down for 8 hours yesterday. The next merged PR should trigger it. @SiddheshKukade create another PR to be sure.

**Comment by SiddheshKukade** [comment]
@palisadoes Sir, Sorry for the delayed response, I was having exams for the past 2 weeks & I didn't got notified about this on GitHub notifications.

I can see that the path still was not updated from the previous PR. (the link is unchanged)
![image](https://user-images.githubusercontent.com/65951872/231710737-e309bf68-087d-47f7-beff-7179315a2a18.png)

Please take a look I think the workflow was not updating the docs.
I've made another PR which will resolve the image issue: https://github.com/PalisadoesFoundation/talawa/pull/1777
Please take a look.

**Comment by palisadoes** [comment]
@literalEval is working on a PR to fix the automated talwawa-docs update

**Comment by SiddheshKukade** [comment]
@palisadoes Sir, the image works fine now.  We can now close this issue.
![image](https://user-images.githubusercontent.com/65951872/235819918-04fe1ec0-43b3-49a0-ae1d-1bbece531cee.png)
